FFXIVClientStructs/FFXIV/Client/System/Input/SoftKeyboards/SteamGamepadTextInput.cs

```diff
@@ -0,0 +1,9 @@
+namespace FFXIVClientStructs.FFXIV.Client.System.Input.SoftKeyboards;
+
+[StructLayout(LayoutKind.Explicit, Size = 0x18)]
+public unsafe partial struct SteamGamepadTextInput {
+    [FieldOffset(0x00)] public SoftKeyboardDevice* PlatformInputMethod;
+
+    [FieldOffset(0x08)] public byte Enabled;
+    [FieldOffset(0x10)] public void* TargetSoftKeyboardInputInterface; // populated as part of 140577A90
+}
```

Comment:
Rename this struct to `SteamGamepadSoftKeyboard`, rename `TargetSoftKeyboardInputInterface` to `TargetInputInterface` and set concrete type.